src/LoginFormContainer.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { useDispatch, useSelector } from 'react-redux';
+
+import { requestLogin, changeLoginField } from './actions';
+
+import { get } from './utils';
+
+import LoginForm from './LoginForm';
+
+export default function LoginFormContainer() {
+  const dispatch = useDispatch();
+
+  const { email, password } = useSelector(get('loginFields'));
+
+  function handleChange({ name, value }) {
+    dispatch(changeLoginField({ name, value }));
+  }
+
+  function handleSubmit() {
+    dispatch(requestLogin());
+  }
+
+  return (
+    <LoginForm
+      fields={{ email, password }}
```

Comment:
```suggestion
      fields={loginFields}
```

받는 쪽에서 구조분해 할당을 처리하면 되기 때문에 요렇게 하면 편하고 표현력도 좋겠네요.